src/projects/fabric.js

```diff
@@ -65,26 +65,17 @@ export const Fabric = () => {
 
       <Subheading>Resources</Subheading>
 
-      <ResponsiveIframe
```

Comment:
to keep things tidy, if the `ResponsiveIframe` component is no longer needed in this file, the statement that imports it should be removed as well.